### DISCUSSION TITLE
Lost access to copilot because of a stupid mistake I made

### DISCUSSION THREAD

**Discussion by Ahmedmma72** [community/community#13580] [Copilot Conversations] [answered]
So this is a very stupid mistake I made. I got the invitation yesterday and everything was perfectly fine but I realized that the GitHub email that I signed for copilot with was an old GitHub email that I no longer use. Stupid me thought that it was the email attached with GitHub that has the access, not the GitHub email itself so I deleted the Github email and attached the email (the one I used to create the Github email that had access) to my current Github email and of course, I no longer have access to copilot. So my question now can I have access again or do I need to wait again. I waited so long to get the invitation to mess everything up in just a couple of hours :(

(ahmedmma726@gmail.com) this is the email that the invitation was sent to and (ahmedmahmoud726) was the GitHub username

**Comment by davecheney** [comment, marked as answer]
Please check your email.

**Reply by Ahmedmma72** [reply]
wow I didn't expect to gain access again thank you very much

**Comment by rafapoerner82** [comment]
Boa noite! Tive que desinstalar meu vscode e ao reinstalar perdi o acesso ao Github Copilot. Poderiam me ajudar?

**Comment by rafapoerner** [comment]
Alguém pode me ajudar? Perdi meu acesso pois formatei meu notebook e ao instalar tudo vi que tinha perdido. Como faço pra reaver meu acesso?
